src/lottie-player.ts

```diff
@@ -165,6 +191,25 @@ const _downloadFile = (fileName: string, blob: Blob) => {
   document.body.removeChild(link);
 }
 
+let _initStatus = InitStatus.IDLE;
+const _initModule = async (engine: Renderer) => {
+  if (engine === Renderer.SW) {
+    return;
+  }
+
+  while (_initStatus === InitStatus.REQUESTED) {
+    await _wait(100);
+  }
+
+  if (_initStatus === InitStatus.INITIALIZED) {
+    return;
+  }
+
+  _initStatus = InitStatus.REQUESTED;
+  await _module.initWgInstance();
```

Comment:
`await` here is necessary?
`await _module.init(engine);`

Reply:
Yes, whatever it calls WebGPU request in function, turns into asynchronous function. The `await` is required.